gwvolman/tasks.py

```diff
@@ -360,6 +360,11 @@ def build_tale_image(task, tale_id, force=False):
         current=1, forceFlush=True)
 
     tale = task.girder_client.get('/tale/%s' % tale_id)
+    while tale["status"] != TaleStatus.READY:
```

Comment:
Any use in setting some maximum time to wait? In the unlikely chance a tale is stuck in a preparing state this would never exit.